packages/forge2d/lib/src/dynamics/world.dart

```diff
@@ -514,7 +515,7 @@ class World {
 
   final Island island = Island();
   final List<Body> stack = [];
-  final Timer broadphaseTimer = Timer();
+  final Timer _broadphaseTimer = Timer();
```

Comment:
Is there a point in making this private? No idea if it ever would be helpful to not have it private, but we should make all fields private that does not need to be public in here at the same time when we do it, so that it's consistent. It would also result in another breaking change to make it private.

Reply:
> Is there a point in making this private?

Oh, I didn't realize that it can still be public, even if `Timer` is not exported. I'll leave it public for now. We'll probably have to go through the whole package at some point and decide what really need to be public.